Source/Core/VideoCommon/AsyncRequests.cpp

```diff
@@ -51,6 +52,7 @@ void AsyncRequests::PushEvent(const AsyncRequests::Event& event, bool blocking)
 
 	if (blocking)
 	{
+		RunGpu();
```

Comment:
yeah, you are going to want to RunGpu even when the request isn't blocking. 
There is still work for the gpu to do.

Otherwise the gpu thread does nothing until some other work triggers it, then it will do this async request first.